src/common/btree.c

```diff
@@ -1336,8 +1336,6 @@ btr_probe(struct btr_context *tcx, dbtree_probe_opc_t probe_opc,
 	bool			 next_level;
 	struct btr_node		*nd;
 	struct btr_check_alb	 alb;
-	struct btr_trace	 traces[BTR_TRACE_MAX];
-	struct btr_trace	*trace = NULL;
 	umem_off_t		 nd_off;
```

Comment:
This code does worry me a little but it definitely was not doing anything useful in its current form other than using stack space